src/css/custom.css

```
@@ -29,3 +29,13 @@
   --docusaurus-highlighted-code-line-bg: rgba(0, 0, 0, 0.3);
   --ifm-background-color: #2a2a40;
 }
+
+[data-theme='dark'] .button.button--secondary:not(#_):not(#_) {
```

Stylelint (17.4.0)

[error] Expected complex :not() pseudo-class notation (selector-not-notation)

(selector-not-notation)